omod-1.8/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_8/ObsResource1_8.java

```diff
@@ -185,11 +187,14 @@ public DelegatingResourceDescription getCreatableProperties() {
 	@Override
 	public Model getGETModel(Representation rep) {
 		ModelImpl model = (ModelImpl) super.getGETModel(rep);
+		DateFormat isoDateFormat = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSSXXX");
+		DateFormat targetDateFormat = new SimpleDateFormat("yyyy-MM-dd");
```

Comment:
I would define these in the utility method (like parseAndFormatDataeProperty).

Reply:
This whole thing should be modified to be included in the existing `value` property getter too and not modify the `getGETModel()`

Reply:
thanks @ibacher if not mistaken you are reffering to `StringProperty()` however this comes from 
`package io.swagger.models.properties;` which caused me to modify **getGETModel()**
Would there be something am missing?

Reply:
@Muta-Jonathan What I meant is that this PR introduces a mechanism for transforming data, but we already have that in the method marked as `@PropertyGetter("value")`, so we should just do the transformation from string -> date type in that function rather than doing this wrapping.